Helper/Order.php

```diff
@@ -338,6 +346,8 @@ public function __construct(
         $this->creditmemoFactory = $creditmemoFactory;
         $this->creditmemoManagement = $creditmemoManagement;
         $this->eventsForThirdPartyModules = $eventsForThirdPartyModules;
+        $this->orderManagement = $orderManagement
+            ?: \Magento\Framework\App\ObjectManager::getInstance()->get(OrderManagementInterface::class);
```

Comment:
Are you sure Magento allows this type of ObjectManager using?
Why we need it?

Reply:
This was done just in case an integration overrides this helper's constructor. Magento does it all the time when introducing new attribute in constructor on class change / update: 

https://github.com/magento/magento2/blob/9544fb243d5848a497d4ea7b88e08609376ac39e/lib/internal/Magento/Framework/App/PageCache/Kernel.php#L84

https://github.com/magento/magento2/blob/89d3cda99975c0db2967165311184be84d26e9b4/lib/internal/Magento/Framework/MessageQueue/Consumer.php#L102

https://github.com/magento/magento2/blob/9544fb243d5848a497d4ea7b88e08609376ac39e/app/code/Magento/PageCache/Controller/Block.php#L59

More: https://github.com/magento/magento2/search?q=%3F%3A+%5CMagento%5CFramework%5CApp%5CObjectManager%3A%3AgetInstance%28%29-%3Eget